BUILD

```diff
@@ -1,4 +1,20 @@
 # Bazel extensions for pybind11
+package(default_visibility = ["//visibility:public"])
+
 licenses(["notice"])
 
 exports_files(["LICENSE"])
+
+# Export `@pybind11` targets in order to make them available to users.
+# In particular, `build_defs.bzl` uses `pybind11` and `msvc_compiler`.
```

Comment:
> In particular, `build_defs.bzl` uses `pybind11` and `msvc_compiler`.

I'm guessing you ran into the problem where a user of pybind11_bazel calls the pybind11_bazel macro, which uses the string `@pybind11//:bla`, but that repo isn't visible to the user's build file? You can work around this by calling `Label()` in the context (bzl file) where the repo is visible.

i.e.

```
PYBIND_COPTS = select({
  Label("@pybind11//:msvc_compiler"): [],
})

PYBIND_DEPS = [
  Label(""@pybind11//:pybind11"),
]
```

The Label() call makes bazel convert the apparent name ("@pybind11") to the canonical name ("@@pybind11~whatever~balbal") (apparent and canonical are the bazel terms used to describe these two different "types" of strings)

I think that makes these aliases unnecessary? Or do some of them still need to be public things users might want to explicitly use themselves for other reasons?

Reply:
TIL! Thank you! :D

I'm going to go ahead and yeet the `pybind11_embed` target **_now_** then, which means that `BUILD` just remains empty.